### PR TITLE
Removing jQuery

### LINKED ISSUE #2215: Update jQuery to 3.5.1 or... Remove jQuery? Discuss!
Removing jQuery from the project has been suggested to me before, and was suggested as a discussion point for the upcoming 8.0 release by @coliff 

I'm open to discussing it. This ticket is the place for that discussion 🤔 😠 😄 😢 

I'm not going to lay out my thoughts just yet, but I'll share what I think are the two different ways to look at it. 

On the one hand- Builtwith's [jQuery Usage Statistics](https://trends.builtwith.com/javascript/jQuery) usage statistics indicate that it's in use on 76% of the top 1m sites. For many people it's still very much a thing. 

On the other hand- For many people who active on GitHub and who stay on the cutting edge of front end development, jQuery is three or four generations old and doesn't have any place at all in their day-to-day development. It may live as a dependency somewhere in some projects, but doing a lot of `$()` stuff is something from the past.

### PR DESCRIPTION
**Pull request by roblarsen**
Closes #2215

(I'll write up a super great commit message when I merge this in)

Additionally, this commit updates Modernizr (I was in the gulpfile and was reminded that they just released) I know! I know. Why not make a big change-set bigger, I always say. 

to do

- [x] Tests! 
- [ ] Write it up! Both cleaning up the docs and writing up the reasoning.

### CODE REVIEW

File: dist/doc/html.md [hunk 1]
```diff
@@ -154,7 +152,33 @@ The only thing Modernizr does regarding polyfills is that the team maintains [a
...
+As of v8.0.0 we no longer include jQuery by default. Web development has
+changed a lot since we started this project and while many millions of sites
+still use jQuery there are many sites and applications that don't. 10 years ago
+jQuery _was_ JavaScript for most developers. That's not the case any more so
+we've made the decidsion to remove jQuery from the project.
+
+If you're intrested in including it, you can easily install jQuery using the
+following command:
+
+``` shell
```
Comment (coliff) [inline dist/doc/html.md:166]: don't think we need `shell` here

File: dist/doc/html.md [hunk 2]
```diff
@@ -154,7 +152,33 @@ The only thing Modernizr does regarding polyfills is that the team maintains [a
...
 polyfills](https://github.com/Modernizr/Modernizr/wiki/HTML5-Cross-Browser-Polyfills).
 
-### jQuery CDN for jQuery
+### jQuery
+
+As of v8.0.0 we no longer include jQuery by default. Web development has
+changed a lot since we started this project and while many millions of sites
+still use jQuery there are many sites and applications that don't. 10 years ago
+jQuery _was_ JavaScript for most developers. That's not the case any more so
+we've made the decidsion to remove jQuery from the project.
```
Comment (coliff) [inline dist/doc/html.md:161]: decision typo

**Comment by roblarsen** [comment]
>- [ ] Maybe put together a little script to add it back in on demand? It wouldn't be that bad. `npm install`, use variations on old gulp tasks to move it to dist and and then add the snippet back in. 🤔 

I'm going to add a couple of gulp tasks for this kind of thing in a separate branch. I'm going to look at the docs again and then merge this in

**Comment by roblarsen** [comment]
Pulling this in since it will end up, from a Docs perspective, being an interim step (more on that soon 🤔 )

**Comment by lovetingyuan** [comment]
wow, awesome

**Pull request merged**